example/bitcoin.js

```diff
@@ -1,24 +1,24 @@
-const ElectrumClient = require("..")
+const ElectrumClient = require('..')
```

Comment:
I ran this test and it seems it's broken?

```
(base) ➜  electrum-client-js git:(websocket) ✗ node example/bitcoin.js 
begin connection: {"host":"electrum.vom-stausee.de","ssl":50002,"tcp":50001,"pruning":null,"http":null,"https":null}
node[8736]: ../src/tcp_wrap.cc:279:static void node::TCPWrap::Connect(const FunctionCallbackInfo<v8::Value> &): Assertion `args[2]->IsUint32()' failed.
 1: 0x10003b540 node::Abort() [/usr/local/bin/node]
 2: 0x10003a638 node::AddEnvironmentCleanupHook(v8::Isolate*, void (*)(void*), void*) [/usr/local/bin/node]
 3: 0x1000dc347 node::TCPWrap::Connect(v8::FunctionCallbackInfo<v8::Value> const&) [/usr/local/bin/node]
 4: 0x1001df32f v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo*) [/usr/local/bin/node]
 5: 0x1001de929 v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<false>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/usr/local/bin/node]
 6: 0x1001de0e5 v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/usr/local/bin/node]
 7: 0x5ffa0f5be3d 
[1]    8736 abort      node example/bitcoin.js
```

Possible that it's a result of npm deps and this new machine I've got. But I did a clone of the base repo, and it seemed to work-

```
begin connection: {"host":"electrum.hsmiths.com","ssl":50002,"tcp":50001,"pruning":null,"http":null,"https":null}
\{ code: 1, message: 'unsupported protocol version: 1.0' }
OnClose:undefined
```

Do we intend to support the old Node.js part of this library, or just want to go full hog on implementing pure browser JS? 😄

Reply:
In the new version, we support node.js and browser. We need to update examples scripts to reflect the improvements we introduced.

Reply:
Created a separate issue for that: https://github.com/nkuba/electrum-client-js/issues/7